conanfile.py

```diff
@@ -19,16 +19,15 @@ def cmake(self):
         if self._cmake is None:
             self._cmake = CMake(self)
             self._cmake.definitions.update({
-                "BUILD_TESTING": False
+                "BUILD_TESTING": True
```

Comment:
We will now switch on running the unit tests as part of the package creation process for CI so we can see the unit-test results there.